packages/@aws-cdk-testing/framework-integ/test/aws-secretsmanager/test/integ.lambda-rotation.ts

```diff
@@ -21,6 +21,7 @@ class TestStack extends cdk.Stack {
         handler: 'index.handler',
         code: lambda.Code.fromInline('NOOP'),
       }),
+      automaticallyAfter: cdk.Duration.hours(4),
```

Comment:
Running the integ test to verify.

Reply:
Hey, I see the integ test fails due to the schedule not working. I still see the schedule is at 30 days.

Reply:
![Uploading Screenshot 2023-12-08 at 10.32.03 AM.png…]()

Reply:
I didn't update `tree.json`, should be good now.